modules/loaders.py

```diff
@@ -147,6 +147,11 @@
         'trust_remote_code',
         'no_use_fast',
         'no_flash_attn',
+    ],
+    'HQQ': [
+        'trust_remote_code',
+        'no_use_fast',
+        'no_flash_attn',
```

Comment:
Are these settings used by the model loader with HQQModelForCausalLM.from_quantized?
It could probably use trust_remote_code=shared.args.trust_remote_code, ditto for the others.

Reply:
removed them as requested